css/style.css

```diff
@@ -0,0 +1,337 @@
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #7d73fc;
+  font-size: 1rem;
+}
+
+.home-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.paragraph {
+  text-align: center;
+  align-items: center;
+  color: #c8c8d4;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  padding: 45px;
+  flex: none;
+}
+
+.home-text h2 {
+  font-size: 1.5rem;
+  line-height: 3rem;
+}
+
+.line1 {
+  width: 500px;
+  top: 9px;
+  border-radius: 10px;
+  padding: 40px;
+}
+
+.third {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  justify-content: space-around;
+  width: 100%;
+  align-items: center;
+}
+
+.list-technologies {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.languages {
+  padding: 15px 1rem 1rem 1rem;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500px;
+  font-size: 12px;
+  line-height: 24px;
+
+  /* padding-top: 15px; */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: #9490ff;
+  letter-spacing: 0.05em;
+}
+
+.menu-language {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500px;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  flex-direction: column;
+  color: #f5f5ff;
+  flex: none;
+  padding: 15px 1rem 1rem 1rem;
+}
+
+.Frameworks {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #9490ff;
+  font-style: normal;
+  font-weight: 500px;
+  font-size: 12px;
+  line-height: 24px;
+
+  /* padding-top: 40px; */
+  justify-self: flex-start;
+  padding: 15px 1rem 1rem 1rem;
+}
+
+.menu-frameworks {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500px;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  flex-direction: column;
+  color: #f5f5ff;
+}
+
+.Skills {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #9490ff;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500px;
+  font-size: 12px;
+  line-height: 24px;
+  padding: 30px;
+  justify-self: flex-start;
+}
+
+.menu-skills {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  flex-direction: column;
+  color: #f5f5ff;
+  padding: 5px;
+}
+
+.line2 {
+  width: 500px;
+  top: 9px;
+  border-radius: 5px;
+  padding: 40px;
+  border: 1px solid #646369;
+}
+
+.socialmedia {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  margin: 0 20 0 0;
+  padding: 5px;
+  justify-content: center;
+  color: #f5f5ff;
+  column-gap: 0.5rem;
+}
+
+.flesh-down {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+}
+
+.My-Recent-Work {
+  font-weight: 300;
+  font-size: 35px;
+  line-height: 45px;
+  align-items: center;
+  display: flex;
```

Comment:
- [ ] The contents of this section should be displayed using the grid i.e. `display: grid`